model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -325,8 +327,8 @@ def lift_up_through(node: Node, reverse_channels: Node):
             node_input_port_0.get_connection().set_source(reverse_channels.out_port(0))
             src.connect(reverse_channels.in_port(0))
             node.out_port(0).get_connection().set_destination(reverse_channels_out_npde.in_port(0))
-            return True
-        return False
+            return True, [reverse_channels_out_npde]
+        return False, []
```

Comment:
When I tried to convert t2t-vit-7.onnx model I received the exception:

```
[ ERROR ]  Exception occurred during running replacer "REPLACEMENT_ID" (<class 'extensions.back.ReverseInputChannels.ApplyReverseChannels'>): Connection has more than one destination: 6
```

So the solution does not look correct.